misc/docs/source/tutorial/code_snippets/quakeml_custom_tags.rst

```diff
@@ -94,3 +99,62 @@ the respective Event type objects (in this example to the Catalog object) as
                               u'value': 'True'},
                 u'my_tag_3': {u'namespace': u'http://some-page.de/xmlns/1.0',
                               u'value': '1'}})
+
+Custom tags can be nested:
+
+.. code-block:: python
+
+    from obspy import Catalog
+    from obspy.core import AttribDict
+
+    ns = 'http://some-page.de/xmlns/1.0'
+
+    my_tag = AttribDict()
+    my_tag.namespace = ns
+    my_tag.value = AttribDict()
+
+    my_tag.value.my_nested_tag1 = AttribDict()
+    my_tag.value.my_nested_tag1.namespace = ns
+    my_tag.value.my_nested_tag1.value = 1.23E+10
+
+    my_tag.value.my_nested_tag2 = AttribDict()
+    my_tag.value.my_nested_tag2.namespace = ns
+    my_tag.value.my_nested_tag2.value = True
+
+    cat = Catalog()
+    cat.extra = AttribDict()
```

Comment:
You could also write `AttribDict(my_tag=my_tag)`; don't know if you want to mention that here.